packages/admin-stories/src/docs/components/Selection/Selection.stories.mdx

```diff
@@ -0,0 +1,111 @@
+
+The second object is the [Selection API](#selection-api).
+
+<Canvas>
+    <Story id="stories-components-selection-useselection-hook--useselection-hook" />
+</Canvas>
+
+### Selection Component
+
+The `Selection` component passes its values to its children via child function.
+
+It provides the same values as `useSelection()`, but the variable names are slightly different:
+
+-   `selectedId` instead of `id`
+-   `selectionMode` instead of `mode`
+-   `selectionApi` contains the [Selection API](#selection-api)
+
+<Canvas>
+    <Story id="stories-components-selection-selection-component--selection-component" />
+</Canvas>
+
+## Selection with URL State
+
+In practice, a common requirement is that each item should have a unique URL.
+For that purpose, the id of the currently selected item has to be written to the URL path.
+Comet Admin has this functionality built-in.
+
+To implement a selection with URL state, you can use the `useSelectionRoute()` hook or the `SelectionRoute` component.
+
+### useSelectionRoute() Hook (Recommended)
+
+The `useSelectionRoute()` hook is similar to `useSelection()`.
+The main difference is that it returns three objects instead of two.
+
+The first - and additional - object is now a `SelectionRoute`.
+This `SelectionRoute` must wrap the selection-specific part of your application for the routing to function correctly.
+
+<Canvas>
+    <Story id="stories-components-selection-useselectionroute-hook--useselectionroute-hook" />
+</Canvas>
+
+### SelectionRoute Component
+
+The `SelectionRoute` is the URL-stateful equivalent of the `Selection` component.
+You can use it exactly like `Selection` since it provides the same values.
+
+<Canvas>
+    <Story id="stories-components-selection-selectionroute-component--selectionroute-component" />
+</Canvas>
+
+## Selection API
+
+The Selection API provides three functions to select and deselect items:
+
+-   `handleSelectId()` for selecting an existing item
+-   `handleAdd()` for "selecting" a new item. It accepts an optional `id` parameter if you generate the new id on the client-side.
+-   `handleDeselect()` for deselection
+
+## Selected
+
+The `Selected` component gives you direct access to the item that matches a selected id.
+
+In the simplest configuration, you have to provide the props `selectedId`, `selectionMode` and `rows`.
+`rows` must be an array containing all items.
+The `Selected` component uses this array to find the selected item based on its id.
+It then provides the item to its children via child function.
+
+<Canvas>
+    <Story id="stories-components-selection-selected--selected" />
+</Canvas>
+
+### Selected with Lazy Loading
+
+If you don't want to preload all item data, you can lazy load each item once it's been selected.
+To do that, you have to provide the two props: `query` and `dataAccessor`.
+
+-   `query`: the GraphQL query to load an item
```

Comment:
What about the variables for this query? Is it always `id`? Should we mention that in the docs?

Reply:
Yes, it's always `id`. I added a sentence to clarify that

> `query`: the GraphQL query to load an item. The query must use the variable `id` for identifying the item

What do you think?

Reply:
Perfect 👍🏼